### DISCUSSION TITLE
Tab crash with empty examples array

### DISCUSSION THREAD

**Discussion by cc-stjm** [jy95/docusaurus-json-schema-plugin#397] [Q&A] [answered]
**Describe the bug**
A schema with an empty "examples" array at the top level leads to a page crash with the error

_Docusaurus error: the `<Tabs>` component requires at least one `<TabItem>` children component_

**To Reproduce**

The following page will demonstrate the crash

```
<JSONSchemaViewer 
    schema={ {
        "$schema": "http://json-schema.org/draft-07/schema#",
        "examples": []
    } } 
    viewerOptions={ 
        {
        "showExamples": true, 
        }
    }
/>
```

Demonstrated with Docusaurus 3.4.0, docusaurus-json-schema-plugin 1.12.1

**Expected behavior**

Behave as if the examples array was not present

**Screenshots**

**Desktop (please complete the following information):**
 - OS:   Served in Linux; viewed in Linux & Windows
 - Browser [e.g. chrome, safari]: Firefox
 - Version [e.g. 22] FIrefox 128

**Comment by jy95** [comment, marked as answer]
Thanks @cc-stjm - should be fixed by 1.12.2.
